miniwob/tests/utils.py

```diff
@@ -0,0 +1,111 @@
+"""Test utilities."""
+
+ALL_TESTABLE_MINIWOB_ENVS = [
```

Comment:
Damn that is alot of environments, is there anyway to make this programmatic? i.e., loop over the environment registry (`gymnasium.registry`) for all env id with `miniwob` in the env_id (more proper would be to check `namespace`)

Reply:
I have changed to reading from the registry.